6home/main.cpp

```diff
@@ -0,0 +1,91 @@
+#include <thread>
+#include <algorithm>
+#include <iostream>
+#include <vector>
+#include <mutex>
+#include <utility>
+#include <cstdlib>
+#include <condition_variable>
+#include <atomic>
+#include <chrono>
+#include <sstream>
+
+
+
+std::mutex m;
+std::condition_variable cv;
+bool canPingWrite {true};
+std::atomic<bool> gameOn {true};
+std::atomic<int> counter {0};
```

Comment:
To wszystko łącznie z funkcjami w klasie zamknij

Reply:
Done